### PR TITLE
feat(camera_view_plugin): add camera view plugin package

### PR DESCRIPTION
**Pull request by Owen-Liuyuxuan**
## Description

In testing the system, sometimes frequently switching from a third-person view following the vehicle and a bird-eye view focusing on the vehicle is needed.

This PR adds two buttons RViz plugin for convenient switching between the two views.

## Related links

https://github.com/autowarefoundation/autoware.universe/pull/5469

## Tests performed

PSim

image
image

## Interface changes

Because operation view only makes sense when it is looking at the vehicle. So in code we force the target frame of operation view  being `base_link` . 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by tkimura4** [comment]
@Owen-Liuyuxuan 
Could you change the file name and plugin name to more generic name?

ex.
operation_view -> third_person_view
route_set_view -> bird_eye_view

**Review by tkimura4 — APPROVED** [review]

**Comment by Owen-Liuyuxuan** [comment]
@tkimura4 Sorry I may have interrupted the auto-merge with a commit to fixing the license problem.

**Comment by shmpwk** [comment]
@Owen-Liuyuxuan 
Could you check https://github.com/autowarefoundation/autoware.universe/actions/runs/6741712068/job/18326863638?pr=5472 ?

File: common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp [hunk 1]
```diff
@@ -0,0 +1,242 @@
+// Copyright 2023-2024 Autoware Foundation
```
Comment (shmpwk) [inline common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp:1]: When publishing a work created in 2023, it is common practice to list only that year. It is not usual to list future years in advance.
Do you have any references for future copyrights?

**Comment by Owen-Liuyuxuan** [comment]
> @Owen-Liuyuxuan Could you check https://github.com/autowarefoundation/autoware.universe/actions/runs/6741712068/job/18326863638?pr=5472 ?

I know that `normalise` should considered as a common spelling error that needs to be fixed. But it seems to be a library function https://www.ogre3d.org/docs/api/1.7/class_ogre_1_1_vector3.html#a9e572a5cc4e1ed826839e5281d8f1d3b

File: common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp (hunk 1 above)
Comment (Owen-Liuyuxuan) [reply common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp]: Thanks for pointing out this fault. It was some naive personal thoughts. I checked some references and I would align to your suggestion.

**Comment by tkimura4** [comment]
The spell-check-partial action succeeded.
https://github.com/autowarefoundation/autoware.universe/actions/runs/6742722495/job/18387566275?pr=5472

**Pull request merged**